scss/base/_typography.scss

```diff
@@ -123,6 +123,12 @@ $body-paragraph--L2: 0.75rem;
 $body-paragraph--L3: 0.5rem;
 $body-paragraph--L4: 0.5rem;
 
+$body-font-weight--emphasized--L0: 500;
+$body-font-weight--emphasized--L1: 500;
+$body-font-weight--emphasized--L2: 500;
+$body-font-weight--emphasized--L3: 500;
+$body-font-weight--emphasized--L4: 500;
+
```

Comment:
This isn't quite correct. you need to add all the variables

```scss
$body-font-family--emphasized--L0: "Haas Grot Text R";
$body-font-family--emphasized--L1: "Haas Grot Text R";
$body-font-family--emphasized--L2: "Haas Grot Text R";
$body-font-family--emphasized--L3: "Haas Grot Text R";
$body-font-family--emphasized--L4: "Haas Grot Text R";

$body-font-size--emphasized--L0: 0rem;
$body-font-size--emphasized--L1: 1.25rem;
$body-font-size--emphasized--L2: 1.0625rem;
$body-font-size--emphasized--L3: 0.9375rem;
$body-font-size--emphasized--L4: 0.8125rem;

$body-font-weight--emphasized--L0: normal;
$body-font-weight--emphasized--L1: 500;
$body-font-weight--emphasized--L2: 500;
$body-font-weight--emphasized--L3: 500;
$body-font-weight--emphasized--L4: 500;

$body-line-height--emphasized--L0: 0rem;
$body-line-height--emphasized--L1: 1.75rem;
$body-line-height--emphasized--L2: 1.5rem;
$body-line-height--emphasized--L3: 1.25rem;
$body-line-height--emphasized--L4: 1.125rem;

$body-paragraph--emphasized--L0: 0rem;
$body-paragraph--emphasized--L1: 1rem;
$body-paragraph--emphasized--L2: 0.75rem;
$body-paragraph--emphasized--L3: 0.5rem;
$body-paragraph--emphasized--L4: 0.5rem;
```

Reply:
This was just copied, but we'd need to fill those out with the correct values for each level